packages/components/organisms/f-form/README.md

```diff
@@ -65,19 +65,51 @@ export default {
 
 ### Props
 
-There may be props that allow you to customise its functionality.
+To add fields to the form and text to the form button a `formData` Object prop should be passed in. The `formData` Object should contain.
+ * **formFields** - ` Array of objects. Object should include
+    * **name** - The field name that data should be stored under
+    * **value** - The value of the field
+    * **translations** - an object contating
```

Comment:
Nit: Text needs a small typo correction.

Reply:
damn 🤦 
Thanks I'll change it now and install a better spell checker for vs code